src/components/Checkout/CheckoutAPICalls.tsx

```diff
@@ -17,9 +17,9 @@ export async function processWelcomeBasket(user: ClientPrincipal | null, current
   return await response.json();
 }
 
-export async function processGeneralItems(user: ClientPrincipal | null, currentUserId: number, checkoutItems: CheckoutItem[]) {
+export async function processGeneralItems(user: ClientPrincipal | null, currentUserId: number, checkoutItems: CheckoutItem[], buildingCode: string) {
   HEADERS['X-MS-API-ROLE'] = getRole(user);
-  const response = await fetch(ENDPOINTS.CHECKOUT_GENERAL_ITEMS, {
+  const response = await fetch(`${ENDPOINTS.CHECKOUT_GENERAL_ITEMS}?buildingCode=${buildingCode}`, {
```

Comment:
Can we move the buildingCode into the JSON body, like the rest? Like this

user_id: currentUserId,
building_code: buildingCode, 
items: checkoutItems.map((item) => ({ id: item.id, quantity: item.quantity })),